packages/ui/FloatingLayer/FloatingLayerDecorator.js

```diff
@@ -1,5 +1,10 @@
 /*
- * Exports the {@link ui/FloatingLayer.FloatingLayerDecorator} higher-order component (HOC).
+ * A higher-order component that adds a FloatingLayer adjacent to wrapped component.
+ *
+ * @class FloatingLayerDecorator
+ * @memberof ui/FloatingLayer.FloatingLayerDecorator
```

Comment:
This is not a jsdoc block and the @memberof is not correct.  Let's remove this change.